src/QueryBuilder.php

```diff
@@ -58,4 +68,187 @@ public function describeTable(string $schemaName, string $tableName): string
             self::quoteIdentifier($tableName)
         );
     }
+
+    public function alterUser(string $userName, array $options): string
+    {
+        return vsprintf(
+            'ALTER USER IF EXISTS 
+            %s
+            SET 
+            ' . self::createQuotedOptionsStringFromArray($options),
+            [
+                self::quoteIdentifier($userName),
+            ]
```

Comment:
tady se motá `.` concat a `sprintf`

Reply:
Je to tady víckrát, přijde mě to hrozně zvláštní.

Reply:
Chtěl jsem právě dodržet, aby to bylo "jakoby" prepared statement. Tj. něco se tam poskládá a pak se k tomu "nabindují" ty věci co je potřeba quotovat. 

Klidně navrhni nějaký jiný způsob. Myslíš, že by jen concat byl lepší? Nebo udělat místo vsprinf naopak concatovat všechno?

Reply:
Já bych udělal všechno `vsprintf` potažmo `sprintf` to už je asi fuk.
```
'ALTER USER IF EXISTS %s SET %s'
```
Ale možná je to zbytečně hnidopišské už.